src/fipsy.hpp

```diff
@@ -77,25 +274,7 @@ class Fipsy {
    */
   void readFeatures(uint32_t& featureRow0, uint32_t& featureRow1, uint16_t& feabits);
 
-  /**
-   * @brief Program fuse table.
-   * @pre enable()
-   */
-  bool program(const FuseTable& fuseTable);
```

Comment:
The `program` function should stay in `Fipsy` class rather than moved to `FuseTable` type.
It needs to verify that fuse table size matches the device ID.